sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -672,7 +673,7 @@ def test_feature_filter_targeting(self, appconfiguration_connection_string):
         sent_config = client.set_configuration_setting(new)
         self._assert_same_keys(sent_config, new)
 
-        assert isinstance(sent_config.filters[0], dict)
+        assert isinstance(sent_config.filters[0], Dict)
```

Comment:
I'm not actually sure if this will work -- `Dict` is preferred for type hints, but in an `isinstance` check like this you'll probably want the implementation type (`dict`)